index.src.html

```diff
@@ -202,33 +202,20 @@
     };
 
     dictionary ClientData {
-        required DOMString       challenge;
-        required DOMString       facet;
-        required JsonWebKey      tokenBinding;
+        required DOMString           challenge;
+        required DOMString           facet;
         required AlgorithmIdentifier hashAlg;
-        WebAuthnExtensions          extensions;
+        JsonWebKey                   tokenBinding;
+        WebAuthnExtensions           extensions;
     };
 
     dictionary WebAuthnExtensions {
     };
 
-    interface AttestationStatement {
-        readonly    attribute AttestationHeader header;
-        readonly    attribute AttestationCore   core;
-        readonly    attribute DOMString         signature;
-    };
-
-    interface AttestationCore {
+    interface WebAuthnAttestation {
```

Comment:
the prior AttestationStatement design mimicked that of JWT/JWS -- we may have a use case for that particular arrangement -- Rolf?  

The header is essentially the "trust" info and the core is the "data" portion.  This proposed re-structuring eliminates those distinctions.